tantar/src/main/java/org/opencadc/tantar/policy/InventoryIsAlwaysRight.java

```diff
@@ -95,25 +95,31 @@ public void resolve(final Artifact artifact, final StorageMetadata storageMetada
             // happen in the case where all files are managed by the inventory but an intervention outside of the
             // Storage Inventory caused a file to disappear.  The file may not have been fully uploaded to begin with
             // either.
-            reporter.report(String.format("Resetting Artifact %s as per policy.", artifact.storageLocation));
+            reporter.report("Resetting Artifact " + artifact.storageLocation + " as per policy.");
 
             validateEventListener.markAsNew(artifact);
         } else if (artifact == null) {
-            reporter.report(String.format("Removing Unknown File %s as per policy.",
-                                          storageMetadata.getStorageLocation()));
+            reporter.report("Removing Unknown File " + storageMetadata.getStorageLocation() + " as per policy.");
 
             validateEventListener.delete(storageMetadata);
         } else {
             // Check metadata for discrepancies.
             if (haveDifferentStructure(artifact, storageMetadata)) {
-                // Then prefer the Artifact.
-                reporter.report(String.format("Replacing File %s as per policy.",
-                                              storageMetadata.getStorageLocation()));
+                // The metadata differs, but for valid reasons (update).
```

Comment:
In either case below (storageMetadata.isValid() true|false) the action is the same: 
```
validateEventListener.delete(storageMetadata);
validateEventListener.markAsNew(artifact);
```
so that a new copy of the file can be synced. It is worth reporting the invalid stored file so that if that happens we'll be able to see evidence of it and try to figure out the cause (in principle, it should "never" happen, but it will...)

Aside: "corrupt" may be too strong a term that makes operators panic :-) "incomplete" or "missing" is really what happens (eg in the file system extended file attributes are used to store the contentChecksum... the most likely case is a filed without that attribute rather than the attribute is there but the value was corrupted. Not saying that corruption is not possible, but I'd not use that term unless we decide to put an error message  into an invalid StorageMetadata and that message says "corrupt" because the adapter can detect that it did set attrs but one or more were actually corrupted.